subscriptions-api.md

```diff
@@ -289,7 +290,35 @@ Each subscription is represented by an object that has the following properties:
   subscription managers. If a filter dialect is specified in a subscription that
   is unsupported by the subscription manager, creation or updates of the
   subscription MUST be rejected with an error. See the
-  [Filter Dialects](#323-filter-dialects) section for further details.
+  [Filters](#323-filters) section for further details.
+- **config** (map) - OPTIONAL. A set of key/value pairs that modify the
+  configuation of the subscription. The `key` MUST be ones of the
+  `subscriptionconfig` keys specified in the Discovery Endpoint Service
+  definition. The `value` MUST conform to the data type specified by the
+  value in the `subscriptionconfig` entry for the `key`.
+
+Below is an example JSON serialization of a susbcription:
+
+```JSON
+{
+  "id": "sub-193-18365",
+  "protocol": "HTTP",
+  "protocolsettings": {
+    "method": "POST"
+  },
+  "sink": "http://example.com/event-processor",
+  "filters": [{
```

Comment:
Originally it was supposed to be
```
"filter":{
  "dialect":"basic",
  "conditions":[
    {
      "type":"prefix",
      "property":"type",
      "value":"com.example"
    }
  ]
}

```

This limits a subscription to the use of a single filter dialect. The fact that there may be multiple `conditions` is specific to the filter dialect `basic`.

Reply:
I'd prefer if we didn't limit it to just one dialect per Subscription. It feels really limiting for a Subscription Manager to support 5 dialects but then the user can only use one at a time.I don't see the benefit to either side of that.

Reply:
Well, it might make it easy to users to create expensive and/or inefficient subscriptions. As an infrastructure developer I usually lean towards being restrictive first and keep the option to broaden the scope later. 😄 
@clemensv What do you think?